en/asgardeo/docs/guides/users/manage-users.md

```diff
@@ -49,6 +49,22 @@ Let's look at how administrators can onboard users from the {{ product_name }} C
     !!! note
         If you have selected **Invite offline** or decided to **set a password for the user**, you can copy the corresponding information at the **Summary** page of the wizard and share with the user.
 
+### Include additional user details
```

Comment:
Having this information here breaks the pattern we have for this section. as this section is for onboarding users. wdyt about having a note here mentioning that if any additional attributes are needed you can do so by following the steps in the configure attributes page and point to that page? In this way we can prevent repetition as well.

Reply:
Fixed with ad1bb28dda3da9ddcbf140c9d1451939a59401bb

Now the section will look as follows.
<img width="1680" alt="Screenshot 2025-04-29 at 16 07 31" src="https://github.com/user-attachments/assets/8b4c34d0-7ece-4ea8-be12-f8754216b902" />